src/main/java/com/wimbli/WorldBorder/WorldFileData.java

```diff
@@ -28,19 +28,30 @@
 	private transient Player notifyPlayer = null;
 	private transient Map<CoordXZ, List<Boolean>> regionChunkExistence = Collections.synchronizedMap(new HashMap<CoordXZ, List<Boolean>>());
 
-	// Use this static method to create a new instance of this class. If null is returned, there was a problem so any process relying on this should be cancelled.
 	public static WorldFileData create(World world, Player notifyPlayer)
+	{
+		return create(world, notifyPlayer, WorldFileDataType.REGION, false);
+	}
+
+	// Use this static method to create a new instance of this class. If null is returned, there was a problem so any process relying on this should be cancelled.
+	// Defaults to "REGION" if "type" is "WorldFileDataType.ALL"
+	public static WorldFileData create(World world, Player notifyPlayer, WorldFileDataType type, boolean silent)
 	{
 		WorldFileData newData = new WorldFileData(world, notifyPlayer);
 
-		newData.regionFolder = new File(newData.world.getWorldFolder(), "region");
+		String subfolder = "region";
```

Comment:
```suggestion
		String subFolder = "region";
```
I'd prefer if this variable was changed to this